### PR TITLE
 Published Customer Support terms and SLA

### PR DESCRIPTION
**Pull request by snikitin-qtl**

### CODE REVIEW

File: docs/sla.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
```
Comment (burtonquantil) [inline docs/sla.md:3]: Start the sentence with "This". 
What is the definition of "Premium Customer"? If defining in another document, should we refer to that other document?

File: docs/sla.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API.
```
Comment (burtonquantil) [inline docs/sla.md:8]: We refer to "portal" in our documentation https://docs.cdnetworks.com/cdn/docs/portal/accessing-portal/logging-in .
Shouldn't we use that term instead?
The first sentence isn't helpful because you define the new phrase using the phrase itself. 
console.cdnetworks.com provides a web UI.   It allows the user to avoid using the API directly, so I'm not sure why you mention it in this context.

File: docs/sla.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API. 
+* "Self-service Documentation" is a self-service documentation portal at https://docs.cdnetworks.com and can be accessed at any time.
```
Comment (burtonquantil) [inline docs/sla.md:9]: Again, defining a term by repeating is not helpful.
I would just say something like "Self-service documentation" is our online help available at https://docs.cdnetworks.com.

File: docs/sla.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+* "Email Support" is a way to obtain help through one of the dedicated email addresses that can be found at https://www.cdnetworks.com/contact
+* "Phone Support" is a way to obtain help through a dedicated phone line. Toll-free phone numbers can be found at https://www.cdnetworks.com/contact.
+* "Direct Tickets Support" is a way to obtain help through a dedicated ticket tracker, where all requests can be managed and handled easily.
+* "Business Hours" are working hours (9AM to 6PM, local time) in California, London, and Beijing.
+* "Business Days" are Monday through Friday at the customer location. 
+* "Urgent requests" are for issues that impact 50% or more of the customer’s users.
```
Comment (burtonquantil) [inline docs/sla.md:15]: The capitalization of these phrases seem arbitrary.

File: docs/sla.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* "High-priority requests" are for issues that impact less than 50% of the customer’s users.
+
+## Standard Support
+
+Trial and Paid customers can use an online self-service resource (console/documentation) provided by the Company and set up services by themselves. If any questions or issues arise, customers can contact Customer Support by email. A response will be available within one business day.
```
Comment (burtonquantil) [inline docs/sla.md:20]: Shouldn't use use the terms defined earlier rather than "console/documentation"?

File: docs/sla.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+## Premium Support
+
+Premium customers can use an online self-service resource (console/documentation) provided by the Company. They can set up services by themselves or use dedicated engineering resources allocated to them by the Company. If any questions or issues arise, premium customers can contact Customer Support by phone, email, and/or Ticket Tracker.
```
Comment (burtonquantil) [inline docs/sla.md:24]: Here, you suddenly introduce "Ticket Tracker" instead of using the previously defined term, ""Direct Tickets Support".

File: docs/sla.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+The Company’s Support Engineers will adhere to the following rules:
+
+Urgent requests:
+1. Engineers will respond to customer within 15 minutes of receipt, start  troubleshooting, and update the status every 30 minutes;
```
Comment (burtonquantil) [inline docs/sla.md:29]: Preface "customer" with "the".
I'm confused about when to use "customer" versus "Customer". You use both in this document.
Remove the extra space character after "start".
Use a period instead of  semicolon to end sentences here and below.
What do you mean by "update the status" ? Do you mean "update the customer with the current status" or "update the ticket's status"?

File: docs/sla.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+2. Engineers will work closely with customers 24x7  until the issue is resolved;
```
Comment (burtonquantil) [inline docs/sla.md:30]: There is also an extra space character after 24x7.

File: docs/sla.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+3. Engineers will issue analysis reports within 24 hours after the incident is resolved.
+
+
+High-priority requests:
+1. Engineers will respond to tickets within 2 hours of receipt;
+2. Engineers will start troubleshooting and update the process every 4 hours;
```
Comment (burtonquantil) [inline docs/sla.md:36]: What is "the process"?   In #1, you refer to "tickets". It's better to stick to one phrase than switching to something new.

File: docs/sla.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+## Service Level Agreement
+
+This Standard Service Level Agreement (the “SLA”) is applicable only to the Service Order Form (“SOF”) that was entered into by and between the CDNetworks entity shown on the SOF (“Company”) and the entity that successfully subscribes to the SOF (“Customer”).
+
+By receiving the Service(s), Customer agrees to and accepts the terms of this SLA. The Effective Date of this SLA is the same date as set forth in the SOF. Company shall provide Customer with the Service(s) specified in an accepted SOF (or set forth in a separate schedule or addendum agreed to and executed by both Parties), subject to the terms of this SLA.
+
+### Service Levels
+
+Company shall at all times perform the Service(s) in accordance with the service level set forth in this Section.
```
Comment (burtonquantil) [inline docs/sla.md:52]: "perform the Service(s)"?  I don't know what that means.

File: docs/sla.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+**SERVICE AVAILABILITY**. The level of availability for Service(s) is 99.9% or per the availability of the origin server, whichever is smaller.
+
+### Measurement Methodology
+
+Terms used herein but not otherwise defined shall have the meanings ascribed to them in the Master Services Agreement. For purposes of this SLA, the following terms have the meanings set forth below:
```
Comment (burtonquantil) [inline docs/sla.md:58]: Where can we see the "Master Services Agreement"?

File: docs/sla.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+Credits will be applied to the Customer’s bill for the Service during the billing cycle following the month in which the credit was granted. The award of service credits is the Customer’s sole remedy and the Supplier’s sole liability for any Service Unavailable Event or loss of service, and shall not exceed the monthly recurring charge paid by Customer for the Services in any event.
+
+This total credit is a sole remedy and not to exceed 100% of the monthly recurring charge of the respective month. The service credits granted on any day shall not exceed the service fee of that affected day in any event.
```
Comment (burtonquantil) [inline docs/sla.md:80]: I am confused by the earlier reference to "50%" and now "100%".

File: docs/sla.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+Credits will be applied to the Customer’s bill for the Service during the billing cycle following the month in which the credit was granted. The award of service credits is the Customer’s sole remedy and the Supplier’s sole liability for any Service Unavailable Event or loss of service, and shall not exceed the monthly recurring charge paid by Customer for the Services in any event.
+
+This total credit is a sole remedy and not to exceed 100% of the monthly recurring charge of the respective month. The service credits granted on any day shall not exceed the service fee of that affected day in any event.				
+### Excluded events
+
+Notwithstanding the foregoing, no service level commitment or service level credit will be made if failure to meet one or more of the above service levels is attributable to any of the following events or circumstances:
```
Comment (burtonquantil) [inline docs/sla.md:83]: You refer to "one or more", but as far as I can see, there is only one defined earlier.

File: docs/support.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
+# Customer Support
+
+In context of CDN360 we consider 3 types of customers.
+
+## Customer Types
+
+1. **Trial Customer** - Customers who onboarded through the Web UI, but have not upgraded to paid customers. These customers have a Trial credit of $50. When they reach $50 consumption, their properties may be undeployed from production automatically. To continue working with the services, an upgrade to Paid Customer is required. No credit card is required to sign-up and start using the Trial account.
```
Comment (burtonquantil) [inline docs/support.md:7]: "paid customers" ,  "Paid Customer".   
Shouldn't we consistently use the defined term?
"sign-up" should be "sign up"

File: docs/support.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+In context of CDN360 we consider 3 types of customers.
+
+## Customer Types
+
+1. **Trial Customer** - Customers who onboarded through the Web UI, but have not upgraded to paid customers. These customers have a Trial credit of $50. When they reach $50 consumption, their properties may be undeployed from production automatically. To continue working with the services, an upgrade to Paid Customer is required. No credit card is required to sign-up and start using the Trial account. 
+
+    One important thing to add is that *Trial Accounts* have limitations. By default it is impossible to deploy to a production environment. We suggest setting up a property, deploying it to the staging environment and testing if it behaves properly. After that it is necessary to contact customer support to enable deployment to production.
+
+
+2. **Paid Customer** - Customers who upgraded through the Web UI using their credit card. These customers are also referred to as standard or month-to-month paid customers. They can stop using service at any time. Paid customers are obliged to pay a minimum amount of $50 per month even if the service consumption is lower than that.
```
Comment (burtonquantil) [inline docs/support.md:12]: stop using "the" service ?

File: docs/support.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+
+3. **Premium Customer** - Paid customers who has a particular monthly commitment and have a long-term contract with us (1Y or longer).
```
Comment (burtonquantil) [inline docs/support.md:15]: "have" not "has" to match the plural "customers".
However, I wonder why you're suddenly making it plural instead of singular since you're defining the term "Premium Customer" not "Premium Customers".
In this context, I'd favor consistently using the singular "customer" throughout this document.

File: docs/support.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Levels of Support
+
+CDN360 offers **Standard** and **Premium** support plans. The Standard plan is offered to Trial Customers and Paid customers.  The Premium plan is offered to Premium Customers only.
+
+|item|Standard Support|Premium Support|
```
Comment (burtonquantil) [inline docs/support.md:22]: Capitalize the column heading "item"

**Review by burtonquantil — CHANGES_REQUESTED** [review]

File: docs/sla.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/sla.md:3]: I agree with Burton about adding "The" to the start of the sentence. Also, if we define "Premium Customer" elsewhere, perhaps this reference should have a link that points users to that location.

File: docs/sla.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/sla.md:8]: Please consider changing the first sentence to: "The Self-service Console at https://console.cdnetworks.com  provides access to self-service features via a Web UI or API." Then we no longer need the second sentence.

File: docs/sla.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/sla.md:9]: I agree with Burton's suggestion.

File: docs/sla.md (hunk 4 above)
Comment (marcfishelquantil) [reply docs/sla.md:15]: We should be consistent with capitalization. Either capitalize "requests" for "Urgent requests" or capitalize only the first letter in the terms surrounds by quotes.
You can remove the comma between "6PM" and "local."
For "Business Days," do we want to mention any holidays when we are not available, such as Xmas?
I suggest changing "Direct Tickets Support" to "Direct Ticket Support" (remove the s from "Tickets"),

File: docs/sla.md (hunk 5 above)
Comment (marcfishelquantil) [reply docs/sla.md:20]: Please write "Paid" as "paid" (no need to capitalize the P).

File: docs/sla.md (hunk 13 above)
Comment (marcfishelquantil) [reply docs/sla.md:83]: No comment.

File: docs/support.md (hunk 14 above)
Comment (marcfishelquantil) [reply docs/support.md:7]: I agree with Burton about capitalization and the use of singular/plural. Please let's be consistent. Here is my suggestion:
"upgraded to paid customer" = "upgraded to Paid Customer"
Also, I agree with Burton about making "sign-up" two words ("sign up").

File: docs/support.md (hunk 15 above)
Comment (marcfishelquantil) [reply docs/support.md:12]: I would reword as follows:

"*Trial Accounts* have limitations that prevent properties from being deployed to a production environment. We suggest that users with Trial Accounts set up a property, deploy it to the staging environment, and then test the property to determine whether it behaves as expected. If it does, contact customer support to allow deployment to production.


2. **Paid customer** - Customers who upgraded through the Web UI using their credit card. These customers are also referred to as standard or month-to-month paid customers. They can stop using the service at any time. Paid customers are obligated to pay a minimum of $50 per month, even if the service consumption fee is lower than this minimum cost."

File: docs/support.md (hunk 17 above)
Comment (marcfishelquantil) [reply docs/support.md:22]: For consistency, please change "and Paid customers" to "and Paid Customers."

**Comment by burtonquantil** [comment]
@ronyayoung  just noticed this very old pull request. 
Do you want to take over on any updates, or is this obsolete?
